Cargo.toml

```diff
@@ -12,10 +12,9 @@ anyhow = "1.0.64"
 bitcoin_hashes = "0.11.0"
 crc = "3.0.0"
 hex = "0.4.3"
+minicbor = { version = "0.18.0", features = ["std"] }
```

Comment:
We could go directly to the `alloc` feature, to anticipate future `no_std` support.

There are only two results whose errors can't implement `std::error::Error` anymore, that need to fixed. They can be wrapped in an `anyhow::Error` like this: `.map_err(|e| anyhow::anyhow!(e)`.

Reply:
Yes, definitely, I was already gonna open an issue about that, `anyhow` is not a good choice for libraries. I would be happy to see a PR!